src/data-mgt/python/devices-tranformation/airqoApi.py

```diff
@@ -88,6 +88,10 @@ def update_sites(self, updated_sites):
             response = self.__request("devices/sites", params, site, "put")
             print(response)
 
+    def refresh_site(self, params):
```

Comment:
this endpoint does not automatically update the `search_name` and `location_name`, you need to explicitly update those two fields using the **Site update** endpoint.

The refresh endpoint only refreshes/updates weather and location data, and NOTHING else. @NoahNsimbe

Reply:
This functionality doesn't update `search_name` and `location_name`. It just refreshes all sites by calling the refresh endpoint as stated in the PR description. To update `search_name` and `location_name`, consider the following documentation. 
https://github.com/airqo-platform/AirQo-api/tree/staging/src/data-mgt/python/devices-tranformation#update-sites-search-names-based-on-csv-file
@Baalmart